test/cypress/integration/05-user_v2.test.js

```diff
@@ -1,26 +1,38 @@
 describe('New users profiles', () => {
   describe('Contributions section', () => {
-    it.skip('Shows contributions with date since and amount contributed', () => {
-      // TODO
+    before(() => {
+      cy.visit('/xdamman/v2');
+      cy.get('[data-cy=Contributions]');
+    });
+    it('Shows contributions with date since and amount contributed', () => {
+      cy.get('[data-cy=collectives]').should('have.length.greaterThan', 1);
+      cy.get('[data-cy=date-since]').contains('since');
+      cy.get('[data-cy=amount-contributed]');
     });
 
-    it.skip('Can load more', () => {
+    it('Can load more', () => {
       // TODO
     });
 
-    it.skip('Can filter by contribution type (admin, financial...etc)', () => {
-      // TODO
+    it('Can filter by contribution type (admin, financial...etc)', () => {
+      cy.get('button[data-cy=filters]').contains(/(Core|Financial|Events)/);
     });
   });
 
   describe('Transactions section', () => {
-    it.skip('Can filter by expense/contributions', () => {
-      // TODO
+    before(() => {
+      cy.visit('/xdamman/v2');
+    });
+    it('Can filter by expense/contributions', () => {
+      cy.get('[data-cy=navbar-contributions]').contains('Contributions');
+      cy.get('[data-cy=navbar-transactions]').contains('Transactions');
     });
 
-    it.skip('Show transactions with all info and links', () => {
-      // TODO Has link to collective
-      // TODO Has transaction description, date and amount
+    it('Show transactions with all info and links', () => {
+      cy.get('[data-cy=transaction-details]');
```

Comment:
As per https://github.com/opencollective/opencollective-frontend/pull/2668#issuecomment-536454524, we also need to check the content as the DOM can be there while the content inside of it is broken.